spec/controllers/catalog_controller_spec.rb

```diff
@@ -98,4 +98,118 @@
       expect(response.body).to include(druid)
     end
   end
+
+  describe 'PATCH #update' do
+    before do
+      post :create, params: { druid: druid, incoming_version: ver, incoming_size: size, endpoint_name: endpoint_name }
```

Comment:
In general, this is a roundabout way of establishing the precondition for a test.  Just because this is a controller test doesn't mean the setup should be done via requests to the controller, when a factory `create` would suffice.  After all, the thing being tested here is `PATCH :update`, which should be the same even if we removed `POST :create`.  Does that make sense?

Reply:
re-pushed w/ changes